docs/ref/contrib/csp/configuration.rst

```diff
@@ -0,0 +1,85 @@
+.. _configuration-chapter:
+
+===============
+Configuring csp
+===============
+
+Content-Security-Policy_ is a complicated header. There are many values
+you may need to tweak here.
```

Comment:
I don't this we can start a security-related documentation pages with "errr it's complicated". The point of the documentation is to teach non-experts, not to scare them.